src/app/features/notifications/model/notifications-activity.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity } from "../../../core/entity/model/entity";
+import { DatabaseField } from "../../../core/entity/database-field.decorator";
+import { DatabaseEntity } from "../../../core/entity/database-entity.decorator";
+
+@DatabaseEntity("Notification")
+export class NotificationActivity extends Entity {
+  @DatabaseField({ dataType: "string" }) title: string;
+  @DatabaseField({ dataType: "string" }) body: string;
+  @DatabaseField({ dataType: "string" }) actionURL: string;
+
+  @DatabaseField({ dataType: "string" }) sentBy: string;
+  @DatabaseField({ dataType: "string" }) fcmToken: string;
+  @DatabaseField({ dataType: "boolean" }) readStatus: boolean;
```

Comment:
You can skip the dataType (just put `@DatabaseField()`) unless it is something special. The annotation infers it automatically from the Typing of the property